LovethTee.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Loveth",
+  "url": "https://github.com/LovethTee",
+  "image": "https://user-images.githubusercontent.com/102753233/181619651-b63b702b-4805-4689-ae46-7110febd5985.jpg",
+  "why": "Tohelp inspire other devs in their fitness journeys",
```

Comment:
Pls separate To & help by a space.
"why": "To help inspire other devs in their fitness journeys"